packages/obonode/obojobo-chunks-multiple-choice-assessment/mc-assessment-explanation.test.js

```diff
@@ -19,6 +19,10 @@ const questionJSON = {
 		solution: {
 			id: 'page-id',
 			type: 'ObojoboDraft.Pages.Page',
+			content: {
```

Comment:
Suggest adding a single test where: you chagne the values of these labels and `expect` them to render the new values.

Reply:
We should probably also have a specific test to make sure the default values are used when not provided.

Reply:
@iturgeon I will look at that